server/ai_process.go

```diff
@@ -85,7 +85,8 @@ type aiRequestParams struct {
 	os          drivers.OSSession
 	sessManager *AISessionManager
 
-	liveParams liveRequestParams
+	liveParams     liveRequestParams
+	mediaMTXClient *media.MediaMTXClient
```

Comment:
We added the `liveParams` struct so we wouldn't keep bloating `aiRequestParams` with live-specific stuff, can we add the mediaMTXClient in there?

Reply:
No worries I've changed this now 👍